CorsixTH/Lua/entities/humanoids/vip.lua

```diff
@@ -265,7 +265,7 @@ function Vip:setVIPRating()
   if count_staff > 1 then
     -- Loop through staff tiredness, if any above verytired, break loop
     for _, staff in ipairs(self.hospital.staff) do
-      if staff.attributes["fatigue"] ~= nil and staff.attributes["fatigue"] >= 0.7 then
+      if staff:getAttribute("fatigue") >= 0.7 then
```

Comment:
Should not be done here, but isn't `verytired` a level config value?

Reply:
`gbv.VeryTired` indeed exists, requires division by 100 (or multiplying fatigue by 100) to use here.
Also if you want to switch to this in future (I would encourage it), please add a default to `base_config.lua` because it's currently missing.

Reply:
Essentially you was a boolean returning function probably e.g.
```lua
function Staff:isVeryTired()
  return staff:getAttribute("fatigue") >= 0.7
end
```
(or replace 0.7 with the gbv.VeryTired -- but you could push that as out of scope)

I reference back to https://github.com/CorsixTH/CorsixTH/pull/2529#issuecomment-2016873106 though regarding receptionist handling.

Reply:
> `gbv.VeryTired` indeed exists, requires division by 100 (or multiplying fatigue by 100) to use here. Also if you want to switch to this in future (I would encourage it), please add a default to `base_config.lua` because it's currently missing.

When someone gets to it, multiplying is the better option.